js/scripts.js

```diff
@@ -0,0 +1,64 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const bookList = document.getElementById('booklist');
+
+const addLi = (title, author) => {
+  const li = document.createElement('li');
+  li.innerText = `${title} - ${author}`;
+  const btn = document.createElement('button');
+  btn.type = 'button';
+  btn.innerText = 'Remove';
+  btn.onclick = 'removeBook';
+  li.appendChild(btn);
+  return li;
+};
+
+const refresh = () => {
+  bookList.innerHTML = '';
+  const storedBooks = JSON.parse(localStorage.books);
+  storedBooks.forEach(
+    (book) => {
+      bookList.appendChild(addLi(book.title, book.author));
+    },
+  );
+};
+
+const addBook = () => {
+  const title = document.getElementById('enterTitle').value;
+  const author = document.getElementById('enterAuthor').value;
+  const storedBooks = JSON.parse(localStorage.books);
+  storedBooks.push(new Book(title, author));
+  localStorage.setItem('books', JSON.stringify(storedBooks));
+  document.getElementById('enterTitle').value = '';
+  document.getElementById('enterAuthor').value = '';
+  refresh();
+};
+
+const removeBook = (e) => {
+  const tgt = e.target;
+  if (tgt.tagName === 'BUTTON') {
+    const title = e.path[1].firstChild.data.split(' - ')[0];
```

Comment:
- [ ] **[Optional] Please be careful while using the `event.path` path method as it might result in compatibility issues.**

For instance, the line of code above will fail in the Firefox browser. Please refer to [this StackOverflow question](https://stackoverflow.com/questions/26195091/determine-event-path-in-dom-event-bubbling) to understand more.